packages/components/src/mobile/alt-text-settings/styles.scss

```diff
@@ -0,0 +1,29 @@
+.altTextEditor {
+	padding: 14px 0;
+	min-height: 80px;
+	max-height: 250px;
```

Comment:
@SiobhyB Is the text area here scrollable upon overflow? If so, I wonder if it'd be simpler to just set a static `height` (perhaps using that 80px value?) to remove some complexity that might come with expanding/shrinking the input?

Reply:
@iamthomasbishop, yes, that's right, the input box currently expands in height when more than four lines of text are added. I've now updated this to a set height of 80px, based on your feedback, so that it no longer expands. :) It's still scrollable when more lines are added.

Reply:
Sounds good! Hopefully that works nicely 😄